conda/recipes/nvstrings/meta.yaml

```diff
@@ -20,22 +18,20 @@ build:
     - CC
     - CXX
     - CUDAHOSTCXX
-    - CMAKE_CXX11_ABI
     - CUDA_VERSION
 
 requirements:
   build:
-    - cmake
-    - python 3.6.* # [py==36]
-    - python 3.7.* # [py==37]
+    - cmake >=3.12
+    - python
   host:
-    - python 3.6.* # [py==36]
-    - python 3.7.* # [py==37]
+    - python
     - setuptools
+    - cmake_setuptools 0.1.3.*
+    - librmm 0.7.*
   run:
-    - python 3.6.* # [py==36]
-    - python 3.7.* # [py==37]
-    - setuptools
+    - python
+    - {{ pin_compatible('librmm') }}
```

Comment:
Do we have a sense of how compatible `librmm` will be? Does it only break things on major releases or minor releases?

Reply:
`librmm` will break on minor versions until version 1.0, but I believe I set a run export for `librmm` to pin to minor version: https://github.com/rapidsai/rmm/blob/branch-0.7/conda/recipes/librmm/meta.yaml#L22

Reply:
Ah sorry, missed that you are also using `run_exports`. That should be sufficient, in which case `pin_compatible` isn't needed here. This will be added in automatically thanks to listing `librmm ` in `host` and using `run_exports` in `librmm`. Hope that makes sense.